modules/manage/partials/recovery-mode.adoc

```diff
@@ -0,0 +1,382 @@
+
+- The following APIs are disabled because some connections, especially malicious ones, can disrupt availability for all users, including admin users:
+** Kafka API (fetch and produce requests)
+** HTTP Proxy
+** Schema Registry
+
+- The following node-wide and cluster-wide processes are disabled as they may disrupt recovery operations:
+** Partition and leader balancers
+** Tiered Storage housekeeping
+** Tiered Storage cache management
+** Compaction
+
+- Redpanda does not load user-managed partitions on disk to prevent triggering partition leadership elections and replication that may occur on startup.
+
+== Prerequisites
+
+You must have the following:
+
+ifdef::env-kubernetes[]
+- A running Redpanda deployment on a Kubernetes cluster.
+- If you are using the Redpanda Helm chart, you need permission to upgrade the Helm release in the namespace where it's deployed.
+- If you are using the Redpanda Operator, you need access to the Redpanda resource manifest.
+endif::[]
+ifndef::env-kubernetes[]
+- xref:get-started:intro-to-rpk.adoc[`rpk`] installed.
+- Local access to machines running Redpanda.
+endif::[]
+
+== Start Redpanda in recovery mode
+
+A broker can only enter recovery mode as it starts up, and not while it is already running. You first set the broker configuration property to enable recovery mode, and then do a broker restart.
+
+ifdef::env-kubernetes[]
+When you enable recovery mode in the Redpanda Helm chart or Redpanda resource, the Helm chart triggers a restart automatically.
+
+. Enable recovery mode:
++
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml,lines=10]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    config:
+      node:
+        recovery_mode_enabled: true
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+
+--
+Helm::
++
+--
+[tabs]
+====
+--values::
++
+.`recovery-mode.yaml`
+[,yaml]
+----
+config:
+  node:
+    recovery_mode_enabled: true
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+--values recovery-mode.yaml --reuse-values
+```
+
+--set::
++
+[,bash]
+----
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --set config.node.recovery_mode_enabled=true
+----
+
+====
+--
+======
+endif::[]
+ifndef::env-kubernetes[]
+. Run the xref:reference:rpk/rpk-redpanda/rpk-redpanda-mode.adoc[`rpk redpanda mode recovery`] command to set the `recovery_mode_enabled` broker configuration property to `true`.
++
+[,bash]
+----
+rpk redpanda mode recovery
+----
++
+Enable recovery mode for all brokers. Although, you can start a mixed-mode cluster, where some brokers are in recovery mode while others are not, it's not recommended.
```

Comment:
is that true @ztlpn ?  I thought its disallowed entirely

Reply:
It was in the brown bag session yesterday that we allow mixed mode.